.ci/common/build.sh

```diff
@@ -50,8 +50,12 @@ build_nvim() {
   echo "Configuring with '${CMAKE_FLAGS}'."
   cmake ${CMAKE_FLAGS} "${TRAVIS_BUILD_DIR}"
 
+  if [[ "${BUILD_32BIT}" == ON ]]; then
+    export CFLAGS="-funsigned-char"
```

Comment:
I'm not sure we can do that everywhere and it should be simple enough to fix the issues once we have visibility into it.